### PR TITLE
Sonar fixes

### PR DESCRIPTION
**Pull request by brsbalazs**
Javítások a statikus analízis alapján.

### CODE REVIEW

**Review by 1m0d — APPROVED** [review]

**Comment by 1m0d** [comment]
great job!

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -14,6 +14,9 @@
 
   private int torpedoCount = 0;
 
+  Random generator = new Random();
```
Comment (1m0d) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: use SecureRandom instead
SecureRandom generator = new SecureRandom();

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +33,17 @@ public TorpedoStore(int numberOfTorpedos){
...
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
     }
 
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
```
Comment (1m0d) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: use var instead of double

**Review by 1m0d — CHANGES_REQUESTED** [review]
rookie mistakes, I was expecting more from you :(

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (brsbalazs) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Has been fixed in  https://github.com/brsbalazs/iet-lab/pull/3/commits/74699656ab1c0bf17d2a9376e2722d1dda1b8e7c

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 2 above)
Comment (brsbalazs) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Find the fiexes in https://github.com/brsbalazs/iet-lab/pull/3/commits/74699656ab1c0bf17d2a9376e2722d1dda1b8e7c

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package hu.bme.mit.spaceship;
 
+import java.security.SecureRandom;
 import java.util.Random;
```
Comment (1m0d) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: useless import

Comment (brsbalazs) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: removed https://github.com/brsbalazs/iet-lab/pull/3/commits/da6a3332285cb5f7e6d79cda3ae19aff5fdaed1f

**Review by 1m0d — APPROVED** [review]

**Pull request merged**